Assets/Tests/InputSystem/Plugins/InputForUITests.cs

```diff
@@ -535,19 +536,34 @@ public void DefaultActions_ShouldNotGenerateAnyVerificationWarnings(bool useProj
         LogAssert.NoUnexpectedReceived();
     }
 
-    [Ignore("We currently allow a PWA asset without an UI action map and rely on defaults instead. This allows users that do not want it or use something else to avoid using it.")]
-    [Test(Description = "Verifies that user-supplied project-wide input actions generates warnings if action map is missing.")]
+    [Test(Description = "Verifies that user-supplied project-wide actions do not generate warnings if action map is missing. We use default actions in this case.")]
     [Category(kTestCategory)]
-    public void ActionsWithoutUIMap_ShouldGenerateWarnings()
+    public void ActionsWithoutUIMap_ShouldNotGenerateWarnings()
```

Comment:
This test makes sense with your changes given that the new desired behaviour is no warnings.